src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Utils/CsvUtil.cs

```diff
@@ -11,6 +14,237 @@ namespace GovUk.Education.ExploreEducationStatistics.Data.Processor.Utils
 {
     public static class CsvUtil
     {
+        /// <summary>
```

Comment:
I think it would be worth adding `#nullable enable` to this class now to catch any problems with all this new code (of which I'm sure there aren't any! 😆 )

Reply:
Already done on the follow-up PR thanks :D